video/out/drm_prime.c

```diff
@@ -68,23 +70,26 @@ int drm_prime_create_framebuffer(struct mp_log *log, int fd,
             }
         }
 
+        if (modifiers[0] && modifiers[0] != DRM_FORMAT_MOD_INVALID) {
+            flags = DRM_MODE_FB_MODIFIERS;
+        }
+
         ret = drmModeAddFB2WithModifiers(fd, width, height, layer->format,
-                                         handles, pitches, offsets,
-                                         modifiers, &framebuffer->fb_id,
-                                         DRM_MODE_FB_MODIFIERS);
-        if (ret < 0) {
+                                         handles, pitches, offsets, modifiers,
+                                         &framebuffer->fb_id, flags);
+        if (ret) {
```

Comment:
fwiw `< 0` isn't incorrect

Reply:
Yep, this change was more about to unify both code paths